### DISCUSSION TITLE
What root mean square value is accepted as high accuracy?

### DISCUSSION THREAD

**Discussion by DaliaA02** [potfit/potfit#113] [Q&A]
Hello, 

I'm using potfit currently to create EAM potential for a certain alloy, for 2404 configurations this is attached my error report, and I will go up to 3000 configurations, what range of values for force root mean square and energy root mean square do I need to have high accuracy? Also, is this root mean square is per atom? Thank you so much for your help! 
![image](https://github.com/user-attachments/assets/77c445d4-c5f6-4266-8ea8-ec655adfc29f)

**Comment by p-bro** [comment]
Hi,

As usual, the answer is “it depends”.  If your reference data is primarily low-temperature, small-strain structures, then a RMS of 0.1 eV/A can be quite significant. On the other hand, if there are lots of liquid configurations at elevated temperature, where forces can be significantly larger than 1 eV/A, then it’s maybe not that bad. Similarly for the energy: 100 meV per atom (yes energies are per atom) can be kind of OK for liquids, but would be not too great for competing crystal structures, for example.

A better way to judge this is to create test data sets (that may or may not be used in fitting), and use those exclusively to get this output. You could then have a test set for zero-T configurations, strained configurations etc. and check how well they are represented (using a test dataset that is not involved in the fitting also helps you guard against overfitting).

Hope that helps,

Peter

--
Dr Peter Brommer
Associate Professor  |  Warwick Centre for Predictive Modelling  |  School of Engineering  |  University of Warwick
***@***.***  |  www.warwick.ac.uk/pbrommer  |  External: +44 (0) 24 7652 8613  |  Internal: 28613

D2.08  |  School of Engineering  |  Library Road  |  Coventry  |  CV4 7AL  |  United Kingdom

PGP Key: 2048R/B6DEA483  |  Fingerprint: 5B6F 9999 79C0 16CD 939E B406 9D2A B6DE A483

I’m a Rainbow Ally, see http://www.warwick.ac.uk/lgbtuasupporters - Pronouns: he/him/his

On 23 Jan 2025, at 00:04, Dalia Ahmed ***@***.***> wrote:



Hello,

I'm using potfit currently to create EAM potential for a certain alloy, for 2404 configurations this is attached my error report, and I will go up to 3000 configurations, what range of values for force root mean square and energy root mean square do I need to have high accuracy? Also, is this root mean square is per atom? Thank you so much for your help!
image.png (view on web)<https://github.com/user-attachments/assets/77c445d4-c5f6-4266-8ea8-ec655adfc29f>

**Reply by DaliaA02** [reply]
By temperature you mean the annealing temperature? Actually it's a medium entropy FCC crystal structure. And by the smal set you mean under a 100 at 0 annealing temperature? Also, if I run it for 3000 configurations and the root mean square is still high what type of improvement I can do? Increasing the number of configurations further? Thank you so much for your help!

**Comment by p-bro** [comment]
Temperature: in this case I meant the temperature the reference configuration was generated at (or is representative for).

If your rms is too high for your applications, this means that potfit cannot find a potential that reproduces your forces adequately well. This could have several reasons:

* There is a good potential, but potfit can’t find it. As the optimisation of a potential is very nonlinear, potfit can get stuck and not find the global optimum. Simulated annealing is supposed to help there, but there is no guarantee. Running potfit from multiple starting potentials might also help.

* The representation of the potential is inadequate. For interpolated potentials that might mean that the sampling points are not in the right places, the cutoff is too short. For analytical potentials it could be that the functions are not adequate. In this case, different functions (or different sampling points) could help.

* The potential form is inadequate: Not all interactions can be perfectly represented by EAM potential (e.g.). Higher-order potentials may help here (e.g. ADP).

Throwing more configurations at the problem does not necessarily help you with finding a potential with lower rms errors (it may help with other issues, e.g. if your potential fit is good, but the potential still does not work in production, as the local environments appearing in production were not adequately represented in the training data).

Without knowing the details of your issue, it’s really difficult to say how and what may be the best way to proceed. But I hope this gives you some leads on how to continue.

Best,

Peter
--
Dr Peter Brommer
Associate Professor  |  Warwick Centre for Predictive Modelling  |  School of Engineering  |  University of Warwick
***@***.***  |  www.warwick.ac.uk/pbrommer  |  External: +44 (0) 24 7652 8613  |  Internal: 28613

D2.08  |  School of Engineering  |  Library Road  |  Coventry  |  CV4 7AL  |  United Kingdom

PGP Key: 2048R/B6DEA483  |  Fingerprint: 5B6F 9999 79C0 16CD 939E B406 9D2A B6DE A483

I’m a Rainbow Ally, see http://www.warwick.ac.uk/lgbtuasupporters - Pronouns: he/him/his

On 23 Jan 2025, at 18:33, Dalia Ahmed ***@***.***> wrote:



By temperature you mean the annealing temperature? Actually it's a medium entropy FCC crystal structure. And by the smal set you mean under a 100 at 0 annealing temperature? Also, if I run it for 3000 configurations and the root mean square is still high what type of improvement I can do? Increasing the number of configurations further? Thank you so much for your help!
